restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/converter/RecordingConverter.java

```diff
@@ -90,7 +93,9 @@ public JsonElement serialize(final Recording recording, final Type type, final J
         writeApiVersion(recording.getApiVersion(), object);
         writeUri(recording.getUri(), object);
         if (recording.getFileUri() != null) {
-            object.addProperty("file_uri", recording.getFileUri().toString());
+            //Previously MybatisRecordingsDao used to prepare absolute URL for fileUri. Now we do it in the converter
+            String fileUri = RestcommConnectServiceProvider.getInstance().uriUtils().resolve(recording.getFileUri(), recording.getAccountSid()).toString();
+            object.addProperty("file_uri", fileUri);
         }
         if (securityLevel.equals(RecordingSecurityLevel.NONE) && recording.getS3Uri() != null) {
             object.addProperty("s3_uri", recording.getS3Uri().toString());
```

Comment:
should we replace hardcoded "s3_uri" with a constant that can be reused across the code?